studio/schemas/documents/blog.ts

```diff
@@ -54,6 +55,16 @@ const blog = defineType({
         }),
       ],
     }),
+    defineField({
```

Comment:
Is this the only way to add static info content to Sanity Studio? With this schema it looks like the blog document has a field storing some dynamic url, while it actually just uses a custom component with static content.

Reply:
We can at least make it `readOnly` to signal that this is not actually a variable property